jax/tools/pgo_nsys_converter.py

```diff
@@ -38,7 +38,14 @@
   profile_folder = os.path.join(os.path.split(args.profile_path)[0], '')
 
   assert isinstance(nsys_path, str)
-  stats_command = [nsys_path, "stats", "--force-overwrite", "true", "--force-export", "true", "--report", "nvtxkernsum", f"{args.profile_path}", "-o", f"{args.pgle_output_path}"]
```

Comment:
Looks like the old report name was nvtxkernsum, but now you're doing nvtx_sum vs nvtxsum. What is the implication of that change?

Reply:
nvtx_kern_sum only provides information about kernel times and does not provide information about memory operations like wrapped copy. With the recent changes in XLA making the requirements for PGO stricter with PGLE, it is an error if the protobuf file has missing instructions (wrapped copy, for example). By using nvtx_sum, we will ensure we get time profile for memory operations too, and this profile gets accepted by XLA.